web/app/components/Chat/Chat.jsx

```diff
@@ -329,21 +329,25 @@ class Chat extends React.Component {
     _onTipSuccess() {
         let tip = this._parseTip();
         let message = {
-            user: "SYSTEM",
+            user: "TROLLBOT",
             message: this.props.currentAccount + " tipped " + tip.to + " " + tip.amount + " " + tip.asset,
             color: "#B71A00"
         };
 
         // Public and local broadcast
-        this._broadCastMessage(message);
+        this._broadCastMessage(message, local = false);
```

Comment:
This will throw an error because `local` is not defined here. The local flag is used in _broadCastMessage to *also* show the message locally, so the tip message is already being broadcast to all other users.